src/App.css

```diff
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: Montserrat, sans-serif;
+}
+
+button {
+  padding: 1rem;
+  background-color: #0290ff;
+  border-radius: 4px;
+  width: 10rem;
+  height: 3rem;
+  color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: larger;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#root {
+  padding: 3rem 5rem;
+  height: max-content;
+}
+
+.app {
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+  border-radius: 4px;
+  padding: 0 0 4rem;
+}
+
+li,
+a {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+nav {
+  display: flex;
+  justify-content: flex-start;
+  gap: 3rem;
+  align-items: center;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  height: 6rem;
+  background-color: #fff;
+}
+
+.nav_links {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  align-items: center;
+  gap: 2.5rem;
+}
+
+.catagories {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 20rem;
+}
+
+.books_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 2rem;
+  padding: 3rem;
+}
+
```

Comment:
I think you forgot to add a line break before then Add book form. As you can see in this image:

Original 
<img width="450" alt="Screen Shot 2022-04-09 at 11 56 32 PM" src="https://user-images.githubusercontent.com/69011963/162588112-beafc1a4-9c0c-471c-9306-b0efa204f17a.png">

Your Design
<img width="450" alt="Screen Shot 2022-04-09 at 11 59 34 PM" src="https://user-images.githubusercontent.com/69011963/162588069-e43c7863-527a-40ec-a256-4b8bf4c5ab34.png">